src/renderer/account/components/Management/Management.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { string, func } from 'prop-types';
+
+import Page from 'shared/components/Page';
+
+import styles from './Management.scss';
+
+export default class Management extends React.PureComponent {
+  static propTypes = {
+    className: string,
+    __progress__: string.isRequired,
+    showErrorToast: func.isRequired
+  };
+
+  static defaultProps = {
+    className: null
+  };
+
+  render() {
+    return (
+      <div className={styles.account}>
+        <Page className={styles.content}>
+          <div> HEY </div>
```

Comment:
HEY

Reply:
WIP code 😄 not yet ready to review haha 
Tijdelijke placeholder tot ik aan de designs begin.
Dit gaat uiteindelijk nooit in de dev branch kopen (door merge commit, zelfde met de vorige comment) dus no worries haha